File: PHP PowerPoint Examples/TextApiDemo/getPptPresentationTextItems.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\TextApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TextApi($configuration);

$name = "PresentationTextItems.pptx";
$withEmpty = 'false'; //the default value is false
$password = null;
$folder = "Powerpoint";
$storage = null;
$result = $apiInstance->getPptPresentationTextItems($name, $withEmpty, $password, $folder, $storage);
print_r($result);

File: Python PowerPoint Examples/TextApiDemo/getPptPresentationTextItems.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.text_api.TextApi(configuration)

#support file format: .pptx/.ppt/.pptm/.pps/.ppsx/.pot/.potx/.odp/
fileName = "PresentationTextItems.pptx"
password = ""
storage = ""
#support :flase/true
with_empty = "false"
inputFolder = "input"
result = api.get_ppt_presentation_text_items(fileName, with_empty=with_empty, password=password, storage=storage, folder=inputFolder)